src/Microsoft.Data.SqlClient/netcore/src/Common/src/Microsoft/Data/Common/AdapterUtil.cs

```diff
@@ -14,6 +14,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Transactions;
+using Microsoft.Data.SqlClient;
```

Comment:
If there is not a serious reason for definition of EventSource inside the **Microsoft.Data.SqlClient** it is better to define this library inside a separate namespace for prevention of type conflict in future.

Reply:
the final goal is to take EventSource out of this namespace and define them in a common folder for both drivers. this is a temp solution just for now.

Reply:
I don't think we intent to change namespace even if we move the class to common. Since the class belongs to the driver, namespace used should not result in conflict, unless there's a usecase that I'm missing out.